src/ansys/mapdl/core/mapdl_geometry.py

```diff
@@ -120,62 +157,428 @@ def _load_iges(self):
+        """Obtain line geometry
+
+        Obtain the active lines as a :class:`pyvista.PolyData` object or
+        a list of :class:`pyvista.PolyData`
+
+        Parameters
+        ----------
+        return_as_list : bool, optional
+            Return the lines in a list, by default False
+
+        Returns
+        -------
+        Union[pv.PolyData, List[pv.PolyData]]
+
+        Examples
+        --------
+
+        Return a single merged mesh.
+
+        >>> line_mesh = mapdl.geometry.get_lines()
+        >>> line_mesh
+        PolyData (0x14917e740)
+          N Cells:    45
+          N Points:   4500
+          N Strips:   0
+          X Bounds:   -1.588e-02, 1.588e-02
+          Y Bounds:   -7.620e-03, 1.778e-02
+          Z Bounds:   -3.180e-03, 1.524e-02
+          N Arrays:   1
+
+        Return a list of lines as indiviudal grids
+
+        >>> lines = mapdl.geometry.get_lines(return_as_list=True)
+        >>> lines
+        [PolyData (0x1492ee7a...rrays:   1, PolyData (0x1491a404...rrays:   1, PolyData (0x1491068c...rrays:   1, PolyData (0x14910662...rrays:   1, PolyData (0x14910632...rrays:   1, PolyData (0x1492eeb0...rrays:   1, PolyData (0x1492eec8...rrays:   1, PolyData (0x1492eee0...rrays:   1, PolyData (0x1492eef8...rrays:   1, PolyData (0x1492ef10...rrays:   1, PolyData (0x1492ef28...rrays:   1, PolyData (0x1492ef40...rrays:   1, PolyData (0x1492ef58...rrays:   1, PolyData (0x1492ef70...rrays:   1, ...]
+
+        """
+        if return_as_list:
+            return self._lines
+        else:
+            return merge_polydata(self._lines)
+
+    @property
+    def areas(self) -> pv.MultiBlock:
+        """Obtain the areas geometry
+
+        Obtain the selected areas as a :class:`pyvista.MultiBlock` object.
+
+        Returns
+        -------
+        pv.MultiBlock
+
+        Examples
+        --------
+
+        >>> mapdl.geometry.areas
+        MultiBlock (0x147ca7640)
+          N Blocks    28
+          X Bounds    -0.016, 0.016
+          Y Bounds    -0.008, 0.018
+          Z Bounds    -0.003, 0.015
+
+        You can plot the entities:
+
+        >>> mapdl.geometry.areas.plot()
+
+        You can access the individual areas using indexing:
+
+        >>> area0 = mapdl.geometry.areas[0]
+        >>> area0
+        UnstructuredGrid (0x147ca4340)
+          N Cells:    10
+          N Points:   18
+          X Bounds:   0.000e+00, 1.588e-02
+          Y Bounds:   -7.620e-03, 1.778e-02
+          Z Bounds:   -3.180e-03, -3.180e-03
+          N Arrays:   3
+
+        Or using the entity name:
+
+        >>> area1 = mapdl.geometry.areas["area 1"]
+        >>> area1
+        UnstructuredGrid (0x147ca4340)
+          N Cells:    10
+          N Points:   18
+          X Bounds:   0.000e+00, 1.588e-02
+          Y Bounds:   -7.620e-03, 1.778e-02
+          Z Bounds:   -3.180e-03, -3.180e-03
+          N Arrays:   3
+
+        You can use this to iterate over the different elements:
```

Comment:
The dreaded "this" pronoun again!

Reply:
![](https://media.giphy.com/media/eP1fobjusSbu/giphy.gif)